interface/main/tabs/css/tabs.css

```diff
@@ -219,7 +219,7 @@ div.closeDlgIframe {
     background: white;
     cursor: pointer;
     margin-top: -0.32%;
-    z-index: 12;
+    z-index: 0;
```

Comment:
I think this is smarter as it will handle all themes and we wont bother checking the others to see if the align or not

Reply:
Will test this in a bit tho

Reply:
Great. Works fine for all the themes tested the thingie (Art's words 😄 )
![screen shot 2018-12-07 at 2 56 14 am](https://user-images.githubusercontent.com/16350814/49622921-ce52a480-f9cb-11e8-806c-8882c61f6add.png)